release-schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "definitions": {
+    "Tender": {
+      "properties": {
+        "reviewDetails": {
+          "title": "Review details",
+          "description": "Precise information concerning the time limits for the review procedures or contact information from which it can be obtained.",
+          "type": [
+            "string",
+            "null"
+          ]
+        },
+        "valueCalculationMethod": {
+          "title": "Value calculation method",
+          "description": "The method used for calculating the estimated value of the concession.",
+          "type": [
+            "string",
+            "null"
+          ]
+        }
+      }
+    },
+    "Organization": {
+      "properties": {
+        "details": {
+          "properties": {
+            "url": {
+              "title": "Website URL",
+              "description": "The Web address of the website of the organization.",
```

Comment:
Use either "The website of the organization." or "The Web address of the organization."

Reply:
:spider_web:

Reply:
Done